Scripts/Commons/common.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
Remove the shebang here so it's apparent this is mean to be `source`d, not ran.

Reply:
This shebang won't be interpreted anyway, because `common.sh` script does not have `x` access permission (it would need to be `chmod +x common.sh` to make it executable), so here it is used for purely informational purposes to make it clear that the script is written in Bash and not some other shell. I'm okay to remove it though, if you prefer to.